src/main/java/com/bmc/truesight/saas/jira/beans/Configuration.java

```diff
@@ -15,7 +15,7 @@
 public class Configuration {
 
     private String jiraHostName;
-    private Integer jiraPort;
+    private String jiraPort;
```

Comment:
We can keep the jiraPort as integer in library, only in case of pluginConfigurationItem we should keep it as String.